tests/k8s_test.go

```diff
@@ -61,54 +62,77 @@ var (
 	clusterName       = "cluster"
 )
 
-func makeCmdWaitUntilStatus(ty, name, status string) string {
-	return fmt.Sprintf("echo 'for i in {1..%d}; do sleep %d; kubectl get --all-namespaces %s %s %s break; done' | bash", k8sRetry, k8sDelaySeconds, ty, name, status)
-}
-
-func makeCmdWaitUntilCreated(ty, name string) string {
-	return makeCmdWaitUntilStatus(ty, name, "&&")
-}
-
-func makeCmdWaitUntilDeleted(ty, name string) string {
-	return makeCmdWaitUntilStatus(ty, name, "||")
-}
-
 func setupFromConfigFile(ty, name string) []helper.Cmd {
 	return []helper.Cmd{
 		{"kubectl create -f " + k8sConfigFile(ty), true},
-		{makeCmdWaitUntilCreated(ty, name), true},
 	}
 }
 
 func tearDownFromConfigFile(ty, name string) []helper.Cmd {
 	return []helper.Cmd{
-		{"kubectl delete -f " + k8sConfigFile(ty), false},
-		{makeCmdWaitUntilDeleted(ty, name), true},
+		{"kubectl delete --grace-period=0 --force -f " + k8sConfigFile(ty), false},
 	}
 }
 
+func makeHasArgsType(ty interface{}, args1 ...interface{}) []interface{} {
+	args := []interface{}{"Manager", "k8s", "Type", ty}
+	args = append(args, args1...)
+	return args
+}
+
+func makeHasArgsNode(node *graph.Node, args1 ...interface{}) []interface{} {
+	m := node.Metadata()
+	args := []interface{}{"Namespace", m["Namespace"], "Name", m["Name"]}
+	args = append(args, args1...)
+	return makeHasArgsType(m["Type"], args...)
+}
+
+func queryNodeCreation(c *CheckContext, query g.QueryString) (node *graph.Node, err error) {
+	err = common.Retry(func() error {
+		const expectedNumNodes = 1
+
+		fmt.Printf("Gremlin Query: %s\n", query)
+		nodes, e := c.gh.GetNodes(query.String())
+		if e != nil {
+			return e
+		}
+
+		if len(nodes) != expectedNumNodes {
+			return fmt.Errorf("Ran '%s', expected %d node, got %d nodes: %+v", query, expectedNumNodes, len(nodes), nodes)
+		}
+
+		if expectedNumNodes > 0 {
+			node = nodes[0]
+		}
+		return nil
+	}, k8sRetry, k8sDelaySeconds*time.Second)
```

Comment:
as k8sDelaySeconds says that it is in second I would put the `8*time.Second` in the constant definition line 47

Reply:
fixed and renamed `k8sDelaySeconds` --> `k8sDelay`